test/c3/appframework/appframework_test.go

```diff
@@ -274,6 +288,11 @@ var _ = Describe("c3appfw test", func() {
 			// Verify RF SF is met
 			testenv.VerifyRFSFMet(deployment, testenvInstance)
 
+			// Verify bundle push status
+			testenv.VerifyClusterMasterBundlePush(deployment, testenvInstance, testenvInstance.GetName(), indexerReplicas, clusterMasterBundleHash)
+			testenv.VerifyDeployerBundlePush(deployment, testenvInstance, testenvInstance.GetName(), indexerReplicas)
```

Comment:
indexerReplicas is representing number of indexers, why are we passing it to Deployer verification method? 
Also re-order verification.